server/model/feed.go

```diff
@@ -17,21 +17,42 @@ package model
 
 // Feed represents an item in the user's feed or timeline.
 type Feed struct {
-	RepoID   int64  `json:"repo_id"                 xorm:"repo_id"`
-	ID       int64  `json:"id,omitempty"            xorm:"pipeline_id"`
-	Number   int64  `json:"number,omitempty"        xorm:"pipeline_number"`
-	Event    string `json:"event,omitempty"         xorm:"pipeline_event"`
-	Status   string `json:"status,omitempty"        xorm:"pipeline_status"`
-	Created  int64  `json:"created,omitempty"       xorm:"pipeline_created"`
-	Started  int64  `json:"started,omitempty"       xorm:"pipeline_started"`
-	Finished int64  `json:"finished,omitempty"      xorm:"pipeline_finished"`
-	Commit   string `json:"commit,omitempty"        xorm:"pipeline_commit"`
-	Branch   string `json:"branch,omitempty"        xorm:"pipeline_branch"`
-	Ref      string `json:"ref,omitempty"           xorm:"pipeline_ref"`
-	Refspec  string `json:"refspec,omitempty"       xorm:"pipeline_refspec"`
-	Title    string `json:"title,omitempty"         xorm:"pipeline_title"`
-	Message  string `json:"message,omitempty"       xorm:"pipeline_message"`
-	Author   string `json:"author,omitempty"        xorm:"pipeline_author"`
-	Avatar   string `json:"author_avatar,omitempty" xorm:"pipeline_avatar"`
-	Email    string `json:"author_email,omitempty"  xorm:"pipeline_email"`
+	RepoID      int64        `json:"repo_id"                     xorm:"repo_id"`
+	ID          int64        `json:"id,omitempty"                xorm:"pipeline_id"`
+	Number      int64        `json:"number,omitempty"            xorm:"pipeline_number"`
+	Event       string       `json:"event,omitempty"             xorm:"pipeline_event"`
+	Status      string       `json:"status,omitempty"            xorm:"pipeline_status"`
+	Created     int64        `json:"created,omitempty"           xorm:"pipeline_created"`
+	Started     int64        `json:"started,omitempty"           xorm:"pipeline_started"`
+	Finished    int64        `json:"finished,omitempty"          xorm:"pipeline_finished"`
+	Branch      string       `json:"branch,omitempty"            xorm:"pipeline_branch"`
+	Ref         string       `json:"ref,omitempty"               xorm:"pipeline_ref"`
+	Refspec     string       `json:"refspec,omitempty"           xorm:"pipeline_refspec"`
+	Deployment  *Deployment  `json:"deployment"                  xorm:"json 'pipeline_deployment'"`
+	PullRequest *PullRequest `json:"pull_request,omitempty"      xorm:"json 'pipeline_pr'"`
+	Release     *Release     `json:"release,omitempty" xorm:"json 'pipeline_release'"`
+	// TODO change json to 'commit' in next major
+	Commit *Commit `json:"commit_pipeline,omitempty"   xorm:"json 'pipeline_commit'"`
```

Comment:
Am I right that we will drop `pipeline_commit` in favor of `commit` in the next major or would we rename again?